manifests/concourse-manifest/concourse-base.yml

```diff
@@ -99,6 +102,7 @@ jobs:
         forward_host: 127.0.0.1
         atc:
           address: 127.0.0.1:8080
+          old_resource_grace_period: 180m
```

Comment:
Is this the reference for 24 hours?

- Added: https://github.com/concourse/concourse/blob/13b003eb5a252801d6abdaa5037c7f6fedbf4f48/docs/release-notes/v0.64.0.any#L6-L57
- Clarified: https://github.com/concourse/concourse/blob/9bb58ca099aab471521edd819584594778e38a5e/docs/release-notes/v0.65.0.any#L13-L14

v0.69.0 says that the latest version of a resource is always kept:

- Release notes: https://github.com/concourse/concourse/blob/9bb58ca099aab471521edd819584594778e38a5e/docs/release-notes/v0.69.0.any#L50-L53
- Tests: https://github.com/concourse/atc/blob/e10b1d7087b6e451deb9475441f074f21ca601e0/lostandfound/baggage_collector_test.go#L418

In which case, do we benefit much from raising the default? I'm inclined to leave it as-is.